src/avt/DBAtts/MetaData/dbatts_exports.h

```diff
@@ -40,11 +40,15 @@
 #define DBATTS_EXPORTS_H
 
 #if defined(_WIN32)
+#if defined(VISIT_BUILD_SHARED_LIBS)
```

Comment:
Here and everywhere else it is used. VISIT_BUILD_SHARED_LIBS isn't used or defined.  We use the standard cmake BUILD_SHARED_LIBS. On by default. Turned off if VISIT_STATIC is on. Please update the files accordingly.

Reply:
Ah, it comes from the integration with ParaView's build system. I'll swap it around with `VISIT_STATIC`.